Packs/GoogleDrive/ReleaseNotes/1_2_2.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Google Drive
+- Fixed an issue where the *parent* argument was sent as a single string instead of a string array in the **google-drive-file-upload** command.
```

Comment:
```suggestion
- Fixed an issue where the *parent* argument was sent as a single string instead of a string array in the ***google-drive-file-upload*** command.
```